vite.config.ts

```diff
@@ -0,0 +1,105 @@
+import { extname, resolve } from "path";
+import { defineConfig, loadEnv, UserConfig } from "vite";
+import viteCompression from "vite-plugin-compression";
+import { nodePolyfills } from "vite-plugin-node-polyfills";
+import {
+  injectCanisterIdPlugin,
+  preRenderAboutPlugin,
+  stripInjectJsScript,
+} from "./vite.plugins";
+
+const defaultConfig = (mode?: string): Omit<UserConfig, "root"> => {
+  const envPrefix = "II_" as const;
+
+  // Expand environment - .env files - with canister IDs
```

Comment:
> Not really, I find quite easy to integrate these .env file. Also I really dislike command line argument so I'm quite happy to not have to care about it. It's also good for the community because sharing simplier command is easier.

Ok, this seems like a personal preference, so maybe we can stick to what we have now? In particular, options like `II_FETCH_ROOT_KEY` really need to be set correctly/have a safe default (of being disabled). In this case it looks like there's no actual default.

The files suggest the command is set, which is not the case (e.g. dev & test builds use different options). I don't mind having the infrastructure for reading those files, but then I would suggest not committing any files (you can add yours to `.git/info/exclude`)

> Canister IDs are injected by a plugin

But why is that mentioned here?

Reply:
Not just a personaly preference, I assume any frontend developer in the world nowadays use `.env`.
Anyway removed then.